apps/builder/app/builder/features/props-panel/props-panel.tsx

```diff
@@ -12,24 +11,55 @@ import {
   ComboboxListboxItem,
   TextField,
   SmallIconButton,
+  Separator,
+  Flex,
+  Text,
+  CSS,
 } from "@webstudio-is/design-system";
 import { ChevronDownIcon, PlusIcon } from "@webstudio-is/icons";
 import type { Publish } from "~/shared/pubsub";
 import { propsStore, useInstanceProps } from "~/shared/nano-states";
-import { CollapsibleSection, ComponentInfo } from "~/builder/shared/inspector";
+import { CollapsibleSection } from "~/builder/shared/inspector";
 import {
   useStyleData,
   type SetProperty as SetCssProperty,
 } from "~/builder/features/style-panel/shared/use-style-data";
 import { renderControl } from "./controls/combined";
 import { usePropsLogic, type NameAndLabel } from "./use-props-logic";
 import { type PropMeta, type PropValue, getLabel } from "./shared";
-
 import { useState } from "react";
+import {
+  getComponentPropsMeta,
+  getComponentMeta,
+  type WsComponentMeta,
+} from "@webstudio-is/react-sdk";
 
 const itemToString = (item: NameAndLabel | null) =>
   item ? getLabel(item, item.name) : "";
 
+const Row = ({ children, css }: { children: React.ReactNode; css?: CSS }) => (
```

Comment:
is there a benefit for using ReactNode instead of JSX.Element? also you probably want to import the specific type {ReactNode}

Reply:
Not sure I understand the problem. `ReactNode` is kinda default type for `children`. So I use it all the time for `children`. For example ` ComponentProps<"div">["children"] === ReactNode`

Reply:
Fixed the import